src/us/wendell/MinecraftManhunt/HuntersCommand.java

```diff
@@ -41,7 +41,7 @@ public HuntersCommand(MinecraftManhunt plugin){
      */
     @Override
     public boolean onCommand(CommandSender commandSender, Command command, String label, String[] args) {
```

Comment:
this whole command organization is just bad lol, I need to delegate set handling to the plugin itself